bundler/lib/bundler/templates/Executable.bundler

```diff
@@ -62,8 +62,9 @@ m = Module.new do
 
   def bundler_requirement
     @bundler_requirement ||=
-      env_var_version || cli_arg_version ||
-        bundler_requirement_for(lockfile_version)
+      env_var_version ||
+      cli_arg_version ||
+      bundler_requirement_for(lockfile_version)
```

Comment:
This seems just personal preference, not RuboCop related, right?

Reply:
Actually, now that the templates are correctly reviewed by RuboCop there’s a `Layout/MultilineOperationIndentation` offense that pops up.

This is what the autocorrect on this file suggests:

```diff
  def bundler_requirement
     @bundler_requirement ||=
       env_var_version || cli_arg_version ||
-        bundler_requirement_for(lockfile_version)
+      bundler_requirement_for(lockfile_version)
   end
```

I thought we could balance this small loss in clarity by splitting this to an extra line. Would RuboCop’s autocorrect suggestion be more in the style of the rest of the project?

Reply:
Aaaah I see! I think from the available alternatives to fix that offense I'd probably go with putting everything in the same line. But your initial edit seems fine too.